analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ObsoleteAttributesNeedExplanation.vb

```diff
@@ -14,6 +14,18 @@ Class Noncompliant
     Private Sub GloballyDeclaredNamespace()
     End Sub
 
+    <Obsolete(Nothing)> ' Noncompliant
+    Sub WithNothing() { }
+    End Sub
+
+    <Obsolete("")> ' Noncompliant
+    Sub WithEmptyString() { }
+    End Sub
+
+    <Obsolete("  ")> ' Noncompliant
+    Sub WithWhiteSpace() { }
+    End Sub
+
```

Comment:
Here too like so.
```vb
<Obsolete("", True)>
<Obsolete("", True, DiagnosticId:="123")>
<Obsolete("", True, DiagnosticId:="123", UrlFormat:="http://docs.domain.com")>
<Obsolete("", DiagnosticId:="123", UrlFormat:="http://docs.domain.com")>
```

Reply:
Added extra testcases, and scenario's resulting in FP, FN, and one resulting in a noncompliant for the wrong reason.